doc/rbd/rbd-snapshot.rst

```diff
@@ -163,29 +188,30 @@ clones rapidly.
            
                Parent                        Child
 
-.. note:: The terms "parent" and "child" refer to a Ceph block device snapshot (parent),
-   and the corresponding image cloned from the snapshot (child). These terms are
-   important for the command line usage below.
+.. note:: The terms "parent" and "child" refer to a Ceph block device snapshot
+   (parent) and the corresponding image cloned from the snapshot (child).
+   These terms are important for the command line usage below.
    
 Each cloned image (child) stores a reference to its parent image, which enables
-the cloned image to open the parent snapshot and read it.   
+the cloned image to open the parent snapshot and read it.
 
-A COW clone of a snapshot behaves exactly like any other Ceph block device
-image. You can read to, write from, clone, and resize cloned images. There are
-no special restrictions with cloned images. However, the copy-on-write clone of
-a snapshot depends on the snapshot, so you **MUST** protect the snapshot before
-you clone it. The following diagram depicts the process.
+A COW (copy-on-write) clone of a snapshot behaves exactly like any other Ceph
+block device image. You can read to, write from, clone, and resize cloned
+images. There are no special restrictions with cloned images. However, the
+copy-on-write clone of a snapshot depends on the snapshot, so you **MUST**
```

Comment:
I don't think "must" needs to be capitalized here.  RBD won't let users to create a clone of a snapshot that isn't protected one way or another.

Reply:
I'm not sure I'm following, what was the ask?

Reply:
@idryomov: I was going to ask you "Do we really need this bolded all-caps MUST in this text?"

But instead of asking, I thought I would wait to see if you complained about it. An opinion freely offered is worth much more than one solicited. I was pleased to see that you also thought that the MUST must go.